package.json

```diff
@@ -2,14 +2,14 @@
   "name": "@nasa-jpl/stellar",
   "description": "A CSS implementation of the Stellar design system.",
   "type": "module",
-  "version": "1.0.28",
+  "version": "1.1.0",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/nasa-jpl/stellar.git"
   },
   "scripts": {
-    "build:css": "rm -rf css && sass --no-source-map scss:css",
+    "build:css": "rm -rf css && sass scss:css",
```

Comment:
I think maybe @duranb was having issues with the source maps on one of his projects so we excluded them from the build? That said source maps would be nice.

Reply:
Oh boy...I don't remember anymore lol. I think I'm ok with this. I'll make my project work with the sourcemaps. That part of the project has halted for the time being anyway.